HW6/yurii_1995/task1.py

```diff
@@ -0,0 +1,30 @@
+# home work 6, task 1
+
+numbers = range(1, 11)
+
+even_numbers = []
+odd_numbers = []
+not_divisible_numbers = []
+
+for number in numbers:
+    if number % 2 == 0:
+        even_numbers.append(number)
+    else:
+        pass
+
+for number in numbers:
+    if number % 3 == 0 and number % 2 != 0:
+        odd_numbers.append(number)
+    else:
+        pass
+
+for number in numbers:
+    if number % 2 != 0 and number % 3 != 0:
+        not_divisible_numbers.append(number)
+    else:
+        pass
+
+print(even_numbers)
+print(odd_numbers)
+print(not_divisible_numbers)
+
```

Comment:
another way:
l1 = [ x for x in range(1, 11) if x % 2 == 0 ]
l2 = [ x for x in range(1, 11) if x % 2 == 1 and x % 3 == 0 ]
l3 = [ x for x in range(1, 11) if not x % 2 == 0 and not x % 3 == 0 ]
print(l1, 'is even multiple of 2')
print(l2, 'is an odd multiple of 3')
print(l3, 'not divisible by 2 and 3')